File: 1935-maximum-number-of-words-you-can-type/1935-maximum-number-of-words-you-can-type.js
/**
 * @param {string} text
 * @param {string} brokenLetters
 * @return {number}
 */
var canBeTypedWords = function (text, brokenLetters) {
  const set = new Set(brokenLetters);
  const spl = text.split(" ");
  let result = 0;

  spl.forEach((word) => {
    let cannotType = false;
    for (let char of word)
      if (set.has(char)) {
        cannotType = true;
        break;
      }
    if (!cannotType) result++;
  });

  return result;
};

File: 1935-maximum-number-of-words-you-can-type/1935-maximum-number-of-words-you-can-type.cs
public class Solution
{
  public int CanBeTypedWords(string text, string brokenLetters)
  {
    int result = 0;
    var charsSet = new HashSet<char>();
    foreach (char c in brokenLetters)
    {
      charsSet.Add(c);
    }
    var spl = text.Split(" ");
    foreach (string word in spl)
    {
      var isValid = true;
      foreach (char c in word)
      {
        if (brokenLetters.Contains(c))
        {
          isValid = false;
          break;
        }
      }
      if (isValid) result++;
    }
    return result;
  }
}